### DISCUSSION TITLE
[Errno 121] Remote I/O error

### DISCUSSION THREAD

**Discussion by ghost** [NVIDIA-AI-IOT/jetbot#418] [Q&A]
We havent used a piOLED display with our i2c converter and even though the convertor is being detected we still get this error " [Errno 121] Remote I/O error " while running basic_motion.ipynb. Is the problem because of not using a piOLED display and is there an alternative?

**Comment by newcgen** [comment]
I using a piOLED display. But i have the same problem. piOLED display works.

**Comment by newcgen** [comment]
Only сhanging the physical address of PCA9685 to 0x60 helped.

**Comment by santaimpersonator** [comment]
It probably can't detect either the camera (also connected through I2C) or motor driver. Without seeing the full error message it would be hard to tell, but it is most likely you motor driver (if the error pops up in the notebook).

Try entering `i2cdetect -y -r 1` in the terminal to list the addresses of all the connected I2C devices to determine if the board can see the motor driver (make sure it is at the correct address). If not, you'll want to check your wiring and test the connections.

**Reply by santaimpersonator** [reply]
Never mind... looks like you figured it out here: https://github.com/NVIDIA-AI-IOT/jetbot/discussions/409
